sdk/boms/azure-external-dependencies-bom/pom.xml

```diff
@@ -0,0 +1,523 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-external-dependencies-bom</artifactId>
```

Comment:
looking at prior art on maven, I think we can change this to "azure-core-dependencies" and drop the 'external' and 'bom'

Reply:
Name changed, also adding another commit trying out adding all track 2 external dependencies into the `azure-sdk-bom`.